extensions/s3/src/main/java/io/deephaven/extensions/s3/S3Instructions.java

```diff
@@ -115,6 +116,12 @@ public Duration readTimeout() {
         return DEFAULT_READ_TIMEOUT;
     }
 
+    /**
+     * The amount of time to wait when writing a fragment before giving up and timing out. If no value has been set then
+     * no timeout value is set in write operations.
+     */
+    public abstract Optional<Duration> writeTimeout();
```

Comment:
```suggestion
    @Default
    public Duration writeTimeout() {
        return DEFAULT_WRITE_TIMEOUT;
    }
```
Let's add a default for write timeout, equal to `DEFAULT_READ_TIMEOUT` to keep the behaviour consistent with before this PR.